frontend/catalyst/jit.py

```diff
@@ -678,7 +680,7 @@ def run(self, args, kwargs):
             Any: results of the execution arranged into the original function's output PyTrees
         """
 
-        results = self.compiled_function(*args, **kwargs)
+        results = self.compiled_function(*(*args, kwargs), {})
```

Comment:
Not sure I fully understand this one? Should not *args, **kwargs work? If not, don't we need ** in front of the kwargs?

Reply:
Actually, after a second look. I could make it work without this change by changing the call funciton of CompiledFunction class.